python/pyspark/sql/readwriter.py

```diff
@@ -770,12 +773,15 @@ def json(self, path, mode=None, compression=None, dateFormat=None, timestampForm
                                 formats follow the formats at ``java.text.SimpleDateFormat``.
                                 This applies to timestamp type. If None is set, it uses the
                                 default value, ``yyyy-MM-dd'T'HH:mm:ss.SSSXXX``.
+        :param lineSep: defines the line separator that should be used for writing. If None is
+                        set, it uses the default value, ``\\n``.
```

Comment:
```
it covers all ``\\r``, ``\\r\\n`` and ``\\n``.
```

Reply:
It is a method of DataFrameWriter. It writes exactly `'\n'`